src/reynir_correct/errfinder.py

```diff
@@ -65,16 +65,29 @@
 from .pattern import PatternMatcher
 
 # Typing stuff
-AnnotationDict = Dict[str, Union[str, int]]
+class AnnotationDict(TypedDict):
+    """The annotation for a sentence"""
+
+    text: str
+    detail: str
+    start: Optional[int]
+    end: Optional[int]
+    original: Optional[str]
+    suggest: Union[str, None]
```

Comment:
Better: `Optional[str]`  (this is equivalent to `Union[str, None]`)

Reply:
Done